src/pages/HomePage/RecentTransactions.tsx

```diff
@@ -27,15 +36,42 @@ type Props = {
 type Ouputs = HomePageTransaction['outputs'];
 
 export function RecentTransactions({ transactions }: Props) {
```

Comment:
Would be nice to use pagination directly on the GraphQL queries as our core has this capability.

If you have the network running locally you can access `http://localhost:4000/playground` to see the GraphQL Playground.

## Examples

### First query without cursor query the latest 5 transactions
```
query {
  transactions(last: 5) {
    pageInfo {
      hasPreviousPage
      hasNextPage
      startCursor
    }
    edges {
      node {
        id
      }
    }
  }
}
```

### Next page query passing the `startCursor`
```
query {
  transactions(last: 5, before: "5#0x8844e7be9a02d03a5681a5b280dc0fc9ac423f50d9c3c9cbfde370d7c11df3e8") {
    pageInfo {
      hasPreviousPage
      hasNextPage
      startCursor
    }
    edges {
      node {
        id
      }
    }
  }
}
```